results/batch_integration_embed/index.qmd

```diff
@@ -2,7 +2,7 @@
 title: "Batch integration embed"
 subtitle: "Removing batch effects while preserving biological variation (embedding output)"
 engine: knitr
-image: featured.png
+image: batch_integration_embed.svg
```

Comment:
Can you rename them all to thumbnail.svg?

```suggestion
image: thumbnail.svg
```

Reply:
Sure! Meant to ask you about that - thanks